src/ILCompiler/src/ILCompiler.csproj

```diff
@@ -26,8 +26,15 @@
     <PackageReference Include="System.CommandLine">
       <Version>0.1.0-e160909-1</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.Build.Framework"> 
```

Comment:
We should not be adding dependency from the ILCompiler to MSBuild. The build task should be in its own assembly.